src/pair_lj_smooth_linear.cpp

```diff
@@ -325,9 +325,9 @@ void PairLJSmoothLinear::read_restart_settings(FILE *fp)
 
 /* ---------------------------------------------------------------------- */
 
-double PairLJSmoothLinear::single(int /*i*/, int /*j*/, int itype, int jtype,
+double PairLJSmoothLinear::single(int i, int j, int itype, int jtype,
```

Comment:
please put the comments back. they reduce warnings when compiling with `-Wall -Wextra`